### PR TITLE
project structure first draft

### PR DESCRIPTION
**Pull request by gorzat**

### CODE REVIEW

**Review by burakukula — APPROVED** [review]

**Review by matzimowski — APPROVED** [review]
🍰

File: Project/structure.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="widtfh=device-width, initial-scale=1">
```
Comment (kdzwinel) [inline Project/structure.html]: `widtfh` 😲

@matzimowski and @burakukula are apparently terrible reviewers  ¯\\\_(ツ)\_/¯

File: Project/structure.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        </section>
+
+        <footer>
+            <p>
+                date info contact
+            </p>
+        </footer>
+    </body>
+
+</html>
```
Comment (kdzwinel) [inline Project/structure.html]: It's a standard to have a newline character at the end of line - https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: Project/structure.html (hunk 1 above)
Comment (matzimowski) [reply Project/structure.html]: 👓

**Review by matzimowski — APPROVED** [review]
looks perfect!

**Review by matzimowski — CHANGES_REQUESTED** [review]
oh no, there is some conflict + some  unnecessary files were commited

**Review by matzimowski — APPROVED** [review]

**Comment by kdzwinel** [comment]
Conflicts!

**Pull request merged**